eng/New-TestResources.ps1

```diff
@@ -10,240 +10,257 @@
 
 [CmdletBinding(DefaultParameterSetName = 'Default', SupportsShouldProcess = $true, ConfirmImpact = 'Medium')]
 param (
-    # Limit $BaseName to enough characters to be under limit plus prefixes, and https://docs.microsoft.com/azure/architecture/best-practices/resource-naming.
```

Comment:
Can we not change the indentation? It will make reviewing diffs every time we sync difficult to spot the relevant bits.